### DISCUSSION TITLE
Whisper returns string of all exclamation points

### DISCUSSION THREAD

**Discussion by BucketofJava** [openai/whisper#865] [General]
I am using the model to evaluate audio generated by a model I am making. However, sometimes, when run the model on my generated audio, it returns '!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!' as the text, and nan as the no_speech_prob. Anyone know why this is?
Thanks so much!

**Comment by AstralScribe** [comment]
It happened with me once, it was some audio encoding issue. You can check whether the audio is encoded in the correct format. I think it happened when I saved using scipy.io.wavfile.write. Another reason could be the sample rate, whisper needs it to be set at 16000.

**Comment by BucketofJava** [comment]
Thank you for your answer! The problem was that my program generating audio occasionally outputted 'nan' as part of the audio, which I fixed with audio[audio!=audio]=0.
